src/app/scripts/cac/control/cac-control-sidebar-directions.js

```diff
@@ -253,6 +261,21 @@ CAC.Control.SidebarDirections = (function ($, Control, BikeModeOptions, Geocoder
         }
     }
 
+    /**
+     * Handles hover events to highlight a given itinerary
+     */
+    function onItineraryHover(event, itinerary) {
+        if (itinerary) {
+            $(options.selectors.itineraryById + currentItinerary.id + '"]')
```

Comment:
I don't love this selector that requires also supplying the closing bracket when using it, particularly because it needs to happen twice here. Maybe a better way to go would be to have a selector for .block-itinerary, and a function getItineraryById that abstracts the dirty work of this query.